packages/core/src/shared/wizards/wizard.ts

```diff
@@ -136,14 +136,23 @@ export class Wizard<TState extends Partial<Record<keyof TState, unknown>>> {
         this._estimator = estimator
     }
 
-    public constructor(private readonly options: WizardOptions<TState> = {}) {
+    public constructor(protected readonly options: WizardOptions<TState> = {}) {
         this.stateController = new StateMachineController(options.initState as TState)
         this.__form = options.initForm ?? new WizardForm()
         this._exitStep =
             options.exitPrompterProvider !== undefined ? this.createExitStep(options.exitPrompterProvider) : undefined
 
         // Subclass constructor logic should live in `init()`, if it exists.
         this._ready = !this.init
+
+        if (typeof this.init === 'function') {
+            // eslint-disable-next-line @typescript-eslint/unbound-method
+            const _init = this.init
```

Comment:
can the eslint ignore be avoided, by something like:

```suggestion
            const _init = this.init.bind(this)
```